### PR TITLE
Refactor site location object map

### PR DESCRIPTION
**Pull request by lbodor**

### CODE REVIEW

File: src/client/app/site-log/site-location.component.ts [hunk 1]
```diff
@@ -192,7 +192,7 @@ export class SiteLocationComponent extends AbstractBaseComponent implements OnIn
             dateInserted: [''],
             deletedReason: ['']
         });
-        this.siteLocation = this.siteLogModel.siteLocation;
+        this.siteLocation = this.siteLogModel.siteInformation.siteLocation;
```
Comment (oehm-smith) [inline src/client/app/site-log/site-location.component.ts]: You can fix up `SiteInfoComponent / moveSiteInformationUp()` now.

File: src/client/app/site-log/site-log.component.ts [hunk 2]
```diff
@@ -394,7 +394,6 @@ export class SiteLogComponent implements OnInit, OnDestroy {
      * @param formValue
      */
     private moveSiteInformationUp(formValue: any) {
-        this.moveSiteInformationUpSpecifically(formValue, 'siteLocation');
```
Comment (oehm-smith) [inline src/client/app/site-log/site-log.component.ts]: Oh here it is!  Great work.

Comment (lbodor) [inline src/client/app/site-log/site-log.component.ts]: Thanks!

**Comment by oehm-smith** [comment]
:+1:  once the tests pass

**Pull request merged**